data/sql/updates/pending_db_world/shandy-fix.sql

```diff
@@ -0,0 +1,27 @@
+
+DELETE FROM creature_text
+WHERE CreatureID = 36856
+  AND GroupID IN (0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
```

Comment:
out the backtick ` in all columns and fields names (same in the bottom SQL query)

Reply:
Backticked